src/buffer/scroll_view.rs

```diff
@@ -991,18 +991,20 @@ fn preview_row<'a>(
                 keyed::Key::Preview(message.hash, idx),
                 button(
                     container(
-                        column![
-                            column![text(title)
+                        column![column![text(title)
+                            .shaping(text::Shaping::Advanced)
+                            .style(theme::text::primary)]
+                        .push_maybe(description.as_ref().map(|description| {
+                            text(description)
                                 .shaping(text::Shaping::Advanced)
-                                .style(theme::text::url)]
-                            .push_maybe(description.as_ref().map(|description| {
-                                text(description)
-                                    .shaping(text::Shaping::Advanced)
-                                    .style(theme::text::primary)
-                            })),
-                            container(image(path).content_fit(ContentFit::ScaleDown))
-                                .max_height(200)
-                        ]
+                                .style(theme::text::secondary)
```

Comment:
Is the text style change here intended?

From url / primary for title / description to primary / secondary

Reply:
Nvm I see it is

Reply:
Yep, it is intended. Looks better.